imgui/core.pyx

```diff
@@ -1374,6 +1374,68 @@ def is_window_collapsed():
     return cimgui.IsWindowCollapsed()
 
 
+def set_tooltip(str text):
+    """Set tooltip under mouse-cursor.
```

Comment:
Sorry for nitpicking but could you add additional new line after first row of docstring. Every other function is documented in style that are roughly compliant to PEP 257 style and it would be great to have this consistent.

BTW. it would be great to have this doctstring summary always fit exactly one line with 79 cols limit (this is actually a comment regarding other functions).